### ISSUE TITLE
JSX element type 'Nouislider' does not have any construct or call signatures.

### ISSUE THREAD

**Issue by aalibaabaa** [mmarkelov/react-nouislider#17]
Hi @mmarkelov ,
first of all, thank you for this package to use nouislider in react apps!

I'm currently facing a problem in my app, when writing typescript.
```javascript
import Nouislider from 'nouislider-react';

function MyApp = () => {
    return (
        <Nouislider .../> // JSX element type 'Nouislider' does not have any construct or call signatures.
    )
}
```

Checking the `index.d.ts` file, it seems that Nouislider typescript declaration expects a "named" export, but indeed it's a default export.

```javascript
export const Nouislider: React.ComponentType<NouisliderProps>;
```

I fixed it in my code this way, but it would be great if you could modify it in your package

```javascript
import Nouislider, { NouisliderProps } from 'nouislider-react';

const Slider = Nouislider as any as (props: NouisliderProps) => JSX.Element;

function MyApp = () => {
    return (
        <Slider .../>
    )
}
```

**Comment by mmarkelov** [comment]
@aalibaabaa thank you for finding it! I does not have much experience with TS :( I'll fix it soon

**Comment by smollweide** [comment]
```
const Nouislider: React.ComponentType<NouisliderProps>;
export default Nouislider;
```

instead of 

```
export const Nouislider: React.ComponentType<NouisliderProps>;
```

Should solve the issue (Named export vs. default export)

**Comment by mmarkelov** [comment]
Should be fixed in new **3.3.3** version

**Comment by aalibaabaa** [comment]
I updated the package in my app and it now works perfect!
Thanks Maksim, for the quick fix!
